temporalio/worker/_workflow_instance.py

```diff
@@ -1954,10 +2011,75 @@ def _enhanced_stack_trace(self) -> temporalio.api.sdk.v1.EnhancedStackTrace:
             )
             return est
 
+    def _temporal_workflow_metadata(self) -> temporalio.api.sdk.v1.WorkflowMetadata:
+        query_definitions = [
+            temporalio.api.sdk.v1.WorkflowInteractionDefinition(
+                name=qd.name if qd.name is not None else "",
```

Comment:
```suggestion
                name=qd.name or "",
```

Pedantic, but this is probably good enough and may improve legibility